Assets/VRM/UniGLTF/Editor/UniGLTFTests.cs

```diff
@@ -150,45 +150,154 @@ public void VersionChecker()
         [Test]
         public void MeshTest()
         {
-            var mesh = new glTFMesh("mesh")
+            var model = new glTFMesh("mesh")
             {
                 primitives = new List<glTFPrimitives>
                 {
                     new glTFPrimitives
                     {
-                        attributes=new glTFAttributes
+                        attributes = new glTFAttributes
                         {
-                            POSITION=0,
+                            POSITION = 0,
                         }
                     }
-                }
+                },
             };
 
-            var f = new JsonFormatter();
-            f.Serialize(mesh);
-
-            var json = new Utf8String(f.GetStoreBytes()).ToString();
+            var json = model.ToJson();
+            Assert.AreEqual(@"{""name"":""mesh"",""primitives"":[{""mode"":0,""indices"":-1,""attributes"":{""POSITION"":0},""material"":0}]}", json);
             Debug.Log(json);
+
+            var c = new JsonSchemaValidationContext("")
+            {
+                EnableDiagnosisForNotRequiredFields = true,
+            };
+            var json2 = JsonSchema.FromType<glTFMesh>().Serialize(model, c);
+            Assert.AreEqual(@"{""name"":""mesh"",""primitives"":[{""mode"":0,""attributes"":{""POSITION"":0},""material"":0,""extensions"":{}}]}", json2);
```

Comment:
`"extensions":{}` may break forward compatibility. Please reference below.
- https://github.com/dwango/UniVRM/pull/123/commits/a5ab17049cae245b52b45ff1aeb93591147922ae#diff-6b022d99473a7681d43bf94e7e94df7fR25
- https://github.com/dwango/UniVRM/pull/123/files#diff-8d8859c465c0b0113b77455f1bf19d89R506

Errors that inform "draco is not supported" might be always happen when UniVRM which have older version loads VRM files which are exported by UniVRM newer than this PR.
`JsonUtility` fills fields not `null` but default initialized values when reading a string like `{}`, Thus inconsistency are happen.

Reply:
Resoleved, https://github.com/dwango/UniVRM/pull/123/commits/38a0a7a715ca282ed4650c02ea8d4248d80ca4ef